app/src/panels/time-series/time-series.vue

```diff
@@ -228,7 +232,12 @@ export default defineComponent({
 				return `${year}-${month}-${day}T${hour}:${minute}:${second}`;
 
 				function padZero(value: number) {
-					return String(value).padStart(2, '0');
+					const intAndDecimal = String(value).split('.');
+
+					if (intAndDecimal.length > 1) {
+						return `${String(intAndDecimal[0]).padStart(2, '0')}.${intAndDecimal[1]}`;
+					}
+					return String(intAndDecimal[0]).padStart(2, '0');
```

Comment:
padZero isn't used in the function above. Does this solve another issue? If so, lets move it into another PR 👍🏻

Reply:
It was an issue where the date strings were returning "invalid date" because of the truncating. I ran into it when I was doing this fix. It might not be an actual issue in production though.